include/lsst/afw/math/ChebyshevBoundedField.h

```diff
@@ -156,6 +156,30 @@ class ChebyshevBoundedField : public table::io::PersistableFacade<ChebyshevBound
                                                       ndarray::Array<double const, 1> const& w,
                                                       Control const& ctrl);
 
+    /**
+     *  Construct a matrix that can be used to fit or evaluate a Chebyshev
+     *  polynomial at a set of points.
+     *
+     *  @param[in]  bbox     Integer bounding box of the resulting approximation.  All
+     *                       given points must lie within lsst::geom::Box2D(bbox).
+     *  @param[in]  x        Array of x coordinate values.
+     *  @param[in]  y        Array of y coordinate values (same size as `x`).
+     *                       noise, w = 1/sigma.
```

Comment:
I don't see where `noise` enters into this.

Reply:
Accidentally copied an extra line.